src/rules/maxLineLengthRule.ts

```diff
@@ -29,12 +36,24 @@ export class Rule extends Lint.Rules.AbstractRule {
             Limiting the length of a line of code improves code readability.
             It also makes comparing code side-by-side easier and improves compatibility with
             various editors, IDEs, and diff viewers.`,
-        optionsDescription: "An integer indicating the max length of lines.",
+        optionsDescription: Lint.Utils.dedent`
+            An integer indicating the max length of lines. Additionally, the \`\"ignore-urls\"\` option
+            can be passed to ignore lines with long URLs.
```

Comment:
do we really want to ignore the line or just subtract the url's length from the line length?
FWIW ESLint ignores the whole line

Reply:
I'm good with either approach. Subtracting line length adds a bit of overhead obviously (there could be multiple URLs on a single line, for instance), but I think it may improve the quality of the rule in general.

Reply:
Adding an option like `"ignore-trailing-comments"` later on will eventually need this logic anyway.
That's why I'm asking for opinions.

Reply:
SGTM--the behavior you're suggesting is probably a more intuitive interpretation of `ignore-urls` anyway, now that I think about it.